### ISSUE TITLE
Jira table template sometimes gets messed up

### ISSUE THREAD

**Issue by vanniktech** [go-jira/jira#176]
I really like the default table template however often times it does not scale well enough and one line is not shown correctly.

Expected:

```
+----------------+------------------------------------------------------------------------------------------------------------------------------------------------+--------------+--------------+--------------+------------+--------------+--------------+
| Issue          | Summary                                                                                                                                        | Type         | Priority     | Status       | Age        | Reporter     | Assignee     |
+----------------+------------------------------------------------------------------------------------------------------------------------------------------------+--------------+--------------+--------------+------------+--------------+--------------+
| foo            |                                                                                                                                                |              |              |              |            |              | 123456789410 |
+----------------+------------------------------------------------------------------------------------------------------------------------------------------------+--------------+--------------+--------------+------------+--------------+--------------+
```

Actual:

```
+----------------+------------------------------------------------------------------------------------------------------------------------------------------------+--------------+--------------+--------------+------------+--------------+--------------+
| Issue          | Summary                                                                                                                                        | Type         | Priority     | Status       | Age        | Reporter     | Assignee     |
+----------------+------------------------------------------------------------------------------------------------------------------------------------------------+--------------+--------------+--------------+------------+--------------+--------------+
| foo            |                                                                                                                                                |              |              |              |            |              | 12345678941011
|
+----------------+------------------------------------------------------------------------------------------------------------------------------------------------+--------------+--------------+--------------+------------+--------------+--------------+
```

This happens on both my Linux as well as on my Mac machine. My hint is that the content of one of the column is too long and there's an off by one error.

**Comment by coryb** [comment]
Yeah, the table configuration is a bit brittle.  It is a bit tricky to wedge it in with the current templating output, but I think we could provide more helper functions template functions to make this better.  I will try to come up with something better.

**Comment by kbakk** [comment]
While the template feature is nice, it would also be good with a simpler way to specify columns to include - eg. something like `jira list --table=Key,Summary,Type,Status,MyField:.customfield_11050`

**Comment by dobbymoodge** [comment]
I've got a PR to attempt to kind-of-fix this in #295. It doesn't address the issue where some values are much longer than a given column assumes, but it does make the column widths easier to tweak, and adjusts some of them to maybe-more-reasonable values. If anyone takes issue with the new values I picked, it will be easy to change since they're [all specified at the top of the template instead of spread throughout](https://github.com/go-jira/jira/pull/295/files#diff-e44675dd1c2a7bb276e0361db0ed8286R296-R312).

It would be nice to be able to grow/shrink column widths according to the longest/shortest value for that column in a range, but that would have to be implemented as a template helper function, since the [go template syntax scopes variables to the control structure they're declared inside of](https://golang.org/pkg/text/template/#hdr-Variables), and we'd have to iterate over `range.issues` to find the field widths. Maybe I can hack on that next.
